homeassistant/components/light/hue.py

```diff
@@ -60,251 +46,142 @@
     'Color temperature light': SUPPORT_HUE_COLOR_TEMP
     }
 
-CONF_ALLOW_IN_EMULATED_HUE = "allow_in_emulated_hue"
-DEFAULT_ALLOW_IN_EMULATED_HUE = True
-
-CONF_ALLOW_HUE_GROUPS = "allow_hue_groups"
-DEFAULT_ALLOW_HUE_GROUPS = True
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Optional(CONF_HOST): cv.string,
-    vol.Optional(CONF_ALLOW_UNREACHABLE): cv.boolean,
-    vol.Optional(CONF_FILENAME): cv.string,
-    vol.Optional(CONF_ALLOW_IN_EMULATED_HUE): cv.boolean,
-    vol.Optional(CONF_ALLOW_HUE_GROUPS,
-                 default=DEFAULT_ALLOW_HUE_GROUPS): cv.boolean,
-})
-
-ATTR_GROUP_NAME = "group_name"
-ATTR_SCENE_NAME = "scene_name"
-SCENE_SCHEMA = vol.Schema({
-    vol.Required(ATTR_GROUP_NAME): cv.string,
-    vol.Required(ATTR_SCENE_NAME): cv.string,
-})
-
 ATTR_IS_HUE_GROUP = "is_hue_group"
 
-CONFIG_INSTRUCTIONS = """
-Press the button on the bridge to register Philips Hue with Home Assistant.
 
-![Location of button on bridge](/static/images/config_philips_hue.jpg)
-"""
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Hue lights."""
+    setup_data(hass)
 
+    for bridge in hass.data[hue.DOMAIN].values():
+        update_lights(hass, bridge, add_devices)
```

Comment:
This cannot work because you throttle `update_lights`, so only the first call will go through.